layouts/community/ergodox/meagerfindings/config.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_MEAGERFINDINGS_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
There is no need for you to undefine a define if your only goal is to redefine it again.

Also just to make your code a bit prettier, some stuff have default defines eg USB_MAX_POWER_CONSUMPTION  has a default value of 500, so no need to define it again:)